casper-v2/src/test/scala/CasperSpec.scala

```diff
@@ -0,0 +1,118 @@
+import cats.effect.Sync
+import cats.syntax.all._
+import coop.rchain.casper.v2.core.Casper.{ConflictScope, FinalizationFringe, MessageScope}
+import coop.rchain.casper.v2.core.syntax.all._
+import coop.rchain.casper.v2.core.{Casper, DependencyGraph, SafetyOracle}
+import monix.eval.Task
+import monix.execution.Scheduler.Implicits.global
+import org.scalatest.{FlatSpec, Matchers}
+import fs2.Stream
+
+class CasperSpec extends FlatSpec with Matchers {
+
+  type Validator = Int
+  type Message   = Int
+  val bonds: Map[Validator, Long] = Map(1 -> 1L, 2 -> 1L, 3 -> 1L)
+
+  // format: off
+  val justificationsMap: Map[Message, List[Message]] = Map(
+                            32 -> List(21, 22, 13), 33 -> List(21, 22, 13),
+    21 -> List(0, 12, 13),  22 -> List(0, 12, 13),
+                            12 -> List(0, 0, 0),    13 -> List(0, 0, 0),
+    0 -> List()
+  )
+  val sendersMap: Map[Message, Validator] = Map(
+                            32 -> 2,                33 -> 3,
+    21 -> 1,                22 -> 2,
+                            12 -> 2,                13 -> 3,
+    0 -> 1
+  )
+  val seqNumsMap: Map[Message, Int] = Map(
+                            32 -> 3,                33 -> 3,
```

Comment:
why `33 -> 3` is `->3` here? Not `2`?

Reply:
Yes, you are right, good catch! Thought it should not influence the test